lua/core/mappings.lua

```diff
@@ -2,8 +2,6 @@ local is_available = astronvim.is_available
 
 local maps = { i = {}, n = {}, v = {}, t = {}, [""] = {} }
```

Comment:
Should `[""] = {}` be removed? I'm not even sure how this is supposed to work. I use <comma> for my leader.

Reply:
Yeah I suppose we can remove this as well since we aren't setting anything with an empty mode